recipes/leptonica/all/conanfile.py

```diff
@@ -85,7 +76,11 @@ def requirements(self):
         if self.options.with_tiff:
             self.requires("libtiff/4.4.0")
         if self.options.with_openjpeg:
-            self.requires("openjpeg/2.5.0")
+            if Version(self.version) >= "1.83.0":
+                # newer leptonica can autodetect any openjpeg version
+                self.requires("openjpeg/2.5.0")
+            else:
+                self.requires("openjpeg/2.4.0")
```

Comment:
It's quite surprising. It was working before this PR with openjpeg 2.5.0 regardless of leptonica version

Reply:
I wondered too. Seems leptonica silently ignored it.

Still fighting with detecting it.

Reply:
I think it can be reverted to openjpeg 2.5.0 unconditionally now? Still don't understand this new branching.

Reply:
Agree, brought it back because it depended on pkgconfig resolution in pre-1.83.0 Leptonica.

Strange build error btw.